src/main/kotlin/pt/up/fe/ni/website/backend/dto/entity/EventDto.kt

```diff
@@ -1,10 +1,13 @@
 package pt.up.fe.ni.website.backend.dto.entity
 
+import com.fasterxml.jackson.annotation.JsonProperty
 import pt.up.fe.ni.website.backend.model.Event
 import java.util.Date
 
 class EventDto(
     val title: String,
     val description: String,
-    val date: Date
+    val date: Date,
+    val teamMembersIds: MutableList<Long>?
```

Comment:
```suggestion
    val teamMembersIds: List<Long>?
```
I didn't check my self but I think this works, right? It's best to not wonder why it's mutable

Reply:
Yes, I had it setup like this from when I was still attempting to use the same MutableList<Account> for the dtos, changed now